tools/marvin/marvin/config/test_data.py

```diff
@@ -775,6 +775,15 @@
         "ostype": "CentOS 5.6 (64-bit)"
 
     },
+    "CentOS6.3template": {
+        "displaytext": "Centos",
+        "name": "Centos",
+        "passwordenabled": False,
+        "ostype": "CentOS 6.3 (64-bit)",
+        "url": "http://10.147.28.7/templates/centos63.ova",
```

Comment:
Can you share or host this ova publicly? Will this test fail for this URL/file not available in one's env.